portality/templates/includes/header-primary-navigation.html

```diff
@@ -1,14 +1,14 @@
 {% set selected_nav = data.nav.entries|selectattr("id", "equalto", section_id)|first %}
-<li class="dropdown primary-nav__item_desktop">
-    <a href="{{ url_for(selected_nav.route) }}" data-toggle="dropdown" class="dropdown__trigger nav-bar__link" id="action-menu-{{ selected_nav.id }}" role="button" aria-haspopup="true" aria-expanded="false">
+<li class="dropdown primary-nav__item_desktop" role="none">
```

Comment:
Idem re: a11y tree

Reply:
It is used in the nav pattern this way on a11y tree here as well:

https://www.digitala11y.com/none-role/

Reply:
Yes, I’ve seen this as well, but we might consider that keeping the unordered list item is something useful for our particular case, in terms of semantics